options/locale/locale_cs-CZ.ini

```diff
@@ -1047,6 +1047,9 @@ pulls.open_unmerged_pull_exists=`Nemůžete provést operaci znovuotevření pro
 pulls.status_checking=Některé kontroly jsou nedořešeny
 pulls.status_checks_success=Všechny kontroly byly úspěšné
 pulls.status_checks_error=Některé kontroly selhaly
+pulls.review_required = Review required
```

Comment:
Do not add translations to other languages, if there is no translation it will fallback to English anyway

Reply:
Got it, thanks for the explanation.